### ISSUE TITLE
Description of each job offer is jumping when collapsing

### ISSUE THREAD

**Issue by ViRPo** [nextcloud/nextcloud.com#493]
I noticed, that when you expand or collapse job position at /jobs, for a short while there is bigger gap between job name and job description while it's expanding or collapsing

It can be seen here: http://recordit.co/ibltd5nPBw
Or you can test it yourself: https://nextcloud.com/jobs/

I failed to find the cause of this, so for now I am submitting it as an issue, however it seems to be connected with class "collapsing"

**Comment by jospoortvliet** [comment]
Yeah, this probably has to do with conflicting javascript from bootstrap and the custom scripts we have :(

@Espina2 am I right?

**Comment by Espina2** [comment]
@jospoortvliet No this is just css.
The problem is on having margins on the paragraphs if you remove it will work just fine.

**Comment by jospoortvliet** [comment]
Ah, that simple huh. @ViRPo wanna try to fix it?

**Comment by Bentong021** [comment]
https://github.com/nextcloud/nextcloud.com/issues/493

**Comment by FWirtz** [comment]
@Espina2 I removed the margins using dev tools - doesn't seem to fix the jumping for me though 😞

**Comment by ViRPo** [comment]
@FWirtz this fixed it for me:
![image](https://user-images.githubusercontent.com/9433048/31225520-c9f717bc-a9d2-11e7-80bc-28c7596838c7.png)
Want to try to fix it inside code?

**Comment by FWirtz** [comment]
... works for me now.
I am pretty sure I did just that though.
Sorry for the confusion.

Yeah, will do! Hope I can make it today, otherwise next week as I'll be away.

**Comment by axlwaii** [comment]
I fixed that [here](https://github.com/nextcloud/nextcloud.com/pull/884) :)
